servers/rendering/renderer_rd/storage_rd/mesh_storage.h

```diff
@@ -615,6 +615,9 @@ class MeshStorage : public RendererMeshStorage {
 	_FORCE_INLINE_ RID multimesh_get_3d_uniform_set(RID p_multimesh, RID p_shader, uint32_t p_set) const {
 		MultiMesh *multimesh = multimesh_owner.get_or_null(p_multimesh);
 		if (!multimesh->uniform_set_3d.is_valid()) {
```

Comment:
I would suggest adding a check for multimesh for null also, eg: `if(multimesh == nullptr) { return RID(); }`

Reply:
Thanks for the feedback Chaos.

Reply:
Done